services/wallet/router/fees.go

```diff
@@ -23,6 +25,9 @@ const (
 	GasFeeHigh
 )
 
+// //////////////////////////////////////////////////////////////////////////////
+// TODO: remove `SuggestedFees` struct once new router is in place
```

Comment:
can we include a GITHUB issue for this TODO - In Mobile that's how we have been doing it as it makes it easier to find out if the relevant issue was dropped or updated in any sense afterwards.